Apache.Druid.Querying.Tests.Unit/DataSerializerShould_ApplyTo.cs

```diff
@@ -70,6 +71,47 @@ public void DimensionMetric()
             .Metric(type => type.Dimension(t));
         Verify(query);
     }
+    
+    private sealed record Activity(
+        [property: DataSourceTimeColumn] DateTimeOffset Timestamp,
+        int Duration,
+        int DomainID,
+        int UserID
+    );
+    private sealed record ActivityDimensions(int DomainID);
+    private sealed record ActivityAggregations(List<long> UserIds, int Duration);
+    
+    [Test]
+    public void ExpressionAggregationWithCombine()
```

Comment:
Let's move this test into `QueryShould.MapToRightJson` or into a completely new file. The tests in `DataSerializerShould_ApplyTo` are meant for making sure `DataSerializer` (serializer for "data" parts of the query, applying  "data conventions" e.g. implicitly converting booleans to numbers) applies to right parts of the query.

Reply:
Thank you for the explanation, I have moved the unit test to the `QueryShould.MapToRightJson` 👌